src/main/java/model/ColorSupplier.java

```diff
@@ -0,0 +1,16 @@
+package model;
+
+public class ColorSupplier {
+    public static String getRandomColor() {
+        String[] colors = new String[]{"acid green", "ash gray", "white",
```

Comment:
we can have less colors, but let's make a `enum Color` and get colors from there